MaritzSTL/koa-ms
# docs: adds typedoc, unstyled eleventy generation

Provides a basic eleventy static site. Integrates with a
typedoc-generated section.

Closes #10
Opens #11 

To test:

- `yarn run docs:serve`
- navigate to localhost:8080
- confirm page1/page2 work
- navigate to `/_api`, ensure you can click around
